python/paddle/v2/fluid/io.py

```diff
@@ -191,6 +192,33 @@ def get_inference_program(target_vars, main_program=None):
     return inference_program
 
 
+def prepend_feed_ops(inference_program, feeded_var_names):
+    global_block = inference_program.global_block()
+    feed_var = global_block.create_var(
+        name='feed', type=core.VarDesc.VarType.FEED_MINIBATCH, persistable=True)
```

Comment:
There might be some problem if fixed the name to `feed`.

Reply:
Will fix this in the next PR.